### ISSUE TITLE
[Request] Show Hints on Startup

### ISSUE THREAD

**Issue by mOctave** [endless-sky/endless-sky#7583]
### Problem Description

There are a lot of secret things you can do in Endless Sky if you know the right keys to press, such as selecting parts of your fleet and giving them special orders, but a lot of the time the controls aren't intuitive and don't get talked about at all in game. All the controls are in the user manual, but the manual isn't packaged with the game and is instead hidden away on an obscure page of the GitHub wiki. This means that many players will never read the manual: the basic controls are in-game, so you don't actually have to read it to understand how to play. When it comes time to do more advanced things, then, many features are hidden from a lot of people.

### Related Issue Links

Related issue about documentation: #3663

### Desired Solution

My solution to this issue is to show hints on startup (even before loading finishes). These would probably have to be pulled from an array in the code itself, since the data files don't all load until loading is done. The message would remain onscreen even after loading finished.

To take this one step further, the player could (or UI changes could) advance the hint to the next one in line, so that the player could see another one of the hints. Hints would only be available on the main menu.
<img width="1382" alt="Screenshot 2022-11-12 at 5 54 54 PM" src="https://user-images.githubusercontent.com/73318970/201501854-480bc613-a749-437f-9be1-30cca1cfc614.png">

### Alternative Approaches

Alternatively, we could:
(a) Bundle a copy of the manual in with the game.
(b) Provide a link to the manual at the top of the credits.txt file:
```
Welcome to Endless Sky!
version 0.9.16

The player manual is available at:
https://github.com/endless-sky/
endless-sky/wiki/PlayersManual

This game is open source.
To report bugs, give feedback,
or contribute to story writing,
artwork, or programming, visit:
- https://endless-sky.github.io
- The game's Steam forums
- Discord (invite: ZeuASSx)

Creator and Lead Programmer
  Michael Zahniser

```
(c) Leave it the way it is, where most people never find out about all the features.

### Additional Context

If this idea is generally approved, I might like to have a go at writing (or helping to write) the code myself, since I've been wanting to learn C++ and contribute to the code side of Endless Sky but haven't had a chance before now.

**Comment by quyykk** [comment]
I personally like the hints during load time better ~~because no one reads the credits~~.

**Comment by Amazinite** [comment]
Closing, because the provide alternative approach of adding a link to the player's manual to the credits was resolved by #8040. Any further requests to add more outside information to the game would probably fall under #3663.
